v19.2/table-expressions.md

```diff
@@ -27,7 +27,7 @@ Parameter | Description
 `name` | One or more aliases for the column names, to use in an [aliased table expression](#aliased-table-expressions).
 `index_name` | Optional syntax to [force index selection](#force-index-selection).
 `func_application` | [Results from a function](#results-from-a-function).
-`preparable_stmt` | [Use the result rows](#using-the-output-of-other-statements) of a [preparable statement](sql-grammar.html#preparable_stmt).
+`row_source_extension_stmt` | [Result rows](#using-the-output-of-other-statements) from a supported [explainable statement](sql-grammar.html#row_source_extension_stmt).
```

Comment:
I have to admit I also don't fully understand the "explainable statement" usage here.  I guess this means "things that return results and are not SET, etc."?  This is a larger issue in our docs than this PR, I think, so I filed https://github.com/cockroachdb/docs/issues/5449.